docs/tutor.rst

```diff
@@ -98,13 +98,24 @@ An optional BRANCH suffix may be appended to the release name to indicate that e
 Contributing to Tutor
 ---------------------
 
-Third-party contributions to Tutor and its plugins are more than welcome! Just make sure to follow these guidelines:
+Contributions to Tutor and its plugins are highly encouraged. Please adhere to the following guidelines:
 
-- Outside of obvious bugs, contributions should be discussed first in the `official Open edX forum <https://discuss.openedx.org>`__.
-- Once we agree on a high-level solution, you should open a pull request on the `Tutor repository <https://github.com/overhangio/tutor/pulls>`__ or the corresponding plugin.
-- Make sure that all tests pass by running ``make test`` (see above).
-- If your PR is in the Tutor core repository, add a changelog entry by running ``make changelog-entry``. Edit the new file and follow the formatting instructions that it contains.
-- Write a good Git commit title and message: explain why you are making this change, what problem you are solving and which solution you adopted. Link to the relevant conversation topics in the forums and describe your use case. We *love* long, verbose descriptions :) As for the title, `conventional commits <https://www.conventionalcommits.org>`__ are preferred. Check the repo history!
+- **General Discussion**: Before addressing anything other than clear-cut bugs, start a discussion on the `official Open edX forum <https://discuss.openedx.org>`__. This facilitates reaching a consensus on a high-level solution.
+- **Pull Requests**: For changes to Tutor core or plugin-specific modifications, open a pull request on the `Tutor repository <https://github.com/overhangio/tutor/pulls>`__.
```

Comment:
... or the corresponding plugin repository.

Reply:
Corrected